addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -18,9 +19,15 @@
      * @private
      * @param {Array<Object>} slideList List of dict reprensenting a slide
      * @param {Object} matcher (see https://underscorejs.org/#matcher)
+     * @param {boolean} returnAll: return all slides matching the criteria instead
+     *     of just the first one (we might have 2 slides with the same slide ID,
+     *     because of "isQuiz" set to True / False)
      */
-    var findSlide = function (slideList, matcher) {
-        var slideMatch = _.matcher(matcher);
+    const findSlide = function (slideList, matcher, returnAll) {
```

Comment:
Can't we simply modify to "findSlides" and change the calls that only want the first one to access the first element of the result?

Reply:
Meh, still strange that a "findSlide" method returns sometimes one, sometimes multiple slides.
It somehow becomes a complicated API for 3 lines of code.

Maybe we could just embed this code and get rid of this discussion? :smile: